Libraries/Opc.Ua.Client/Session.cs

```diff
@@ -4002,7 +4002,7 @@ protected virtual void OnKeepAlive(ServerState currentState, DateTime currentTim
         /// </summary>
         protected virtual bool OnKeepAliveError(ServiceResult result)
         {
-            long delta = DateTime.UtcNow.Ticks - Interlocked.Read(ref m_lastKeepAliveTime);
+            long delta = HiResClock.TickCount - Interlocked.Read(ref m_lastKeepAliveTime);
 
             Utils.LogInfo(
                 "KEEP ALIVE LATE: {0}s, EndpointUrl={1}, RequestCount={2}/{3}",
```

Comment:
this calculation is now based on millisecond counter, not ticks, and needs to be fixed.